docs/whats-new.md

```diff
@@ -52,6 +52,13 @@ Target Release Date: Q1 2022.
 
 - The `WKBWriter` supports options within a `wkb` key, instead of at the top level.
 
+**@loaders.gl/textures**
+- Fixed issues:
+  - Overflow issue on extreme colors in color_distance().
```

Comment:
Are these the detailed release notes from basis? I think we can drop these here and just link to them. They seem to be very specialized.

Reply:
Rmoved